### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by connoro13c**
Removed individual rate limits, and included direction to go to discord to ask for an increase to individual rate limits. 

## Test plan

No review required: deploys tested changes.
derp

### CODE REVIEW

File: doc/app/index.md [hunk 1]
```diff
@@ -56,13 +56,7 @@ If you're on a version that's 2023.6.13 or older, we recommend you uninstall the
...
-
-### Cody Chat
-Interactions with Cody Chat (whether in the app UI or in the editor extension) are capped at 100 requests per day.
-
-### Code autocomplete
-Code autocomplete is capped at 1000 requests per day. Learn more about [Cody Completions](../cody/autocomplete.md).
+There are several forms of rate limiting that help us control costs for free versions of Cody. We expect to relax these limits as we continue development on [Cody Gateway](../cody/explanations/cody_gateway.md). If you hit these limits, you can can request an increase by visiting our [discord](https://discord.gg/C2EBERDZ) channel and asking for a higher limit of both chats and completions. If you'd like to use your own third-party LLM provider instead of Cody Gateway, you must create your own key with Anthropic or OpenAI and [update your app configuration](app_configuration.md).
```
Comment (jdorfman) [inline doc/app/index.md:59]: Good call. I would use https://discord.com/servers/sourcegraph-969688426372825169

cc @MaedahBatool

Comment (taylorsperry) [reply doc/app/index.md:59]: If that's the link that goes to the #app channel in discord, then I agree with that suggestion.

**Review by taylorsperry — APPROVED** [review]

**Review by connoro13c — COMMENTED** [review]
updated discord link per Justin's suggestion

**Comment by kalanchan** [comment]
check CLA

**Comment by kalanchan** [comment]
@cla-bot check

**Review by kalanchan — APPROVED** [review]

**Pull request closed without merging**